### PR TITLE
[CP-WG] Agenda and content for meeting on Feb 22, 2024

### PR DESCRIPTION
**Pull request by AlexHentschel**
updated `core_protocol_working_group/meetings/2024-02-22.md`:
* high-level agenda for meeting on Jan 22, 2024
* Meeting content

updated `core_protocol_working_group/README.md`:
* added organizational details

added `core_protocol_working_group/meetings/yyyy-mm-dd.md`: 
* template for future meetings

### CODE REVIEW

**Comment by AlexHentschel** [comment]
fyi @vishalchangrani @durkmurder if you are interested to take a look (not required)

File: core_protocol_working_group/meetings/yyyy-mm-dd.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Meeting 1
```
Comment (vishalchangrani) [inline core_protocol_working_group/meetings/yyyy-mm-dd.md]: is there a reason to repeat the template here ?

Comment (AlexHentschel) [reply core_protocol_working_group/meetings/yyyy-mm-dd.md]: 💡 thanks for the comment. I overlooked the template file [`meeting_template.md`](https://github.com/onflow/Flow-Working-Groups/blob/main/meeting_template.md) in the root folder. 

Removed my template file [`core_protocol_working_group/meetings/yyyy-mm-dd.md`](https://github.com/onflow/Flow-Working-Groups/pull/9/files/1210294cd2458bc8b4d5f6f99aaa019a6bd502ca#diff-ed896756e2b1a0ce9d61c53bdbf9fb6612117c48e6ef088499732612e6e4f68f) in https://github.com/onflow/Flow-Working-Groups/pull/9/commits/821ca7952453cd5fa49ce5109921c489caf35cc2

File: core_protocol_working_group/meetings/2024-02-22.md [hunk 2]
```diff
@@ -1,25 +1,100 @@
-# Meeting 1
+# Inaugural Meeting [00] of the Core Protocol Working Group
 
-## Date: Feb 22nd, 2024, Thursday, 10-11 AM PST
+**Date**: February 22, 2024 [Thursday], 10-11am PST
```
Comment (franklywatson) [inline core_protocol_working_group/meetings/2024-02-22.md]: Strictly speaking we're on PT right now, but will be back on PST in May

**Review by franklywatson — APPROVED** [review]

File: core_protocol_working_group/meetings/2024-02-22.md (hunk 2 above)
Comment (AlexHentschel) [reply core_protocol_working_group/meetings/2024-02-22.md]: Ah, the time zone stuff is soooo confusing. 
This [website](https://www.timeanddate.com/time/zone/canada/vancouver) lists `PST` [Pacific Standard Time] as the current time zone and `PDT` [Pacific Daylight Time] starting from Sun, Mar 10 at 2:00 am ... hope I am reading that correctly:

> The term Pacific Time (PT) is often used to denote the local time in areas observing either [Pacific Standard Time (PST)](https://www.timeanddate.com/time/zones/pst) or [Pacific Daylight Time (PDT)](https://www.timeanddate.com/time/zones/pdt).
>
> In other words, in locations observing [Daylight Saving Time (DST)](https://www.timeanddate.com/time/dst/) during part of the year, Pacific Time is not static but switches between PST and PDT.

**Comment by vishalchangrani** [comment]
@AlexHentschel -  I updated the 2023-02-20.md file and organized it a little further.

**Comment by vishalchangrani** [comment]
Also, @AlexHentschel - I would love if we also covered during the intro to zero downtime upgrade the following:
1. Why does Flow have to undergo a hard upgrade?
2. Our journey so far - we started with 8 sporks a year and are now down to 4 (one every quarter), and this year the plan is to have only one.

**Comment by AlexHentschel** [comment]
> Also, @AlexHentschel - I would love if we also covered during the intro to zero downtime upgrade the following:
> 1. Why does Flow have to undergo a hard upgrade?
> 2. Our journey so far - we started with 8 sporks a year and are now down to 4 (one every quarter), and this year the plan is to have only one.

added in https://github.com/onflow/Flow-Working-Groups/pull/9/commits/94920c5cd5f3aee1cca1aa87542dba2323029759 (plus some minor formatting updates)

File: core_protocol_working_group/meetings/2024-02-22.md (hunk 2 above)
Comment (franklywatson) [reply core_protocol_working_group/meetings/2024-02-22.md]: My bad, thanks for setting it straight

**Pull request merged**